.gitmodules

```diff
@@ -5,3 +5,6 @@
 	path = src/zlib
 	url = https://github.com/wxWidgets/zlib.git
 	branch = wx
+[submodule "src/expat"]
+	path = src/expat
+	url = https://github.com/wxWidgets/libexpat.git
```

Comment:
Add `branch = wx` here as well?

Reply:
Yes, thanks. I'll update this PR soon, once the CI builds for the png one finish and I can merge that one as they're going to conflict with each other.